Makefile

```diff
@@ -48,16 +47,16 @@ copy_app=rm -rf gaia/outoftree_apps/$(app) && \
 		cp -r apps/$(app) gaia/outoftree_apps/) &&
 
 hellomake:
-	$(build) && (cd gaia && make)
+	$(build) && (cd gaia && make GAIA_DISTRIBUTION_DIR=`pwd`/spark)
 
 shallow:
 	$(build)
 
 install-gaia:
-	$(build) && (cd gaia && make install-gaia)
+	$(build) && (cd gaia && make install-gaia GAIA_DISTRIBUTION_DIR=`pwd`/spark)
```

Comment:
Maybe we should just export this if it's unset instead, so that we get it for free everywhere that we call ```make``` and it's not overriding any custom settings.